products/jbrowse-cli/test/data/volvox.ix

```diff
@@ -0,0 +1,1425 @@
+1 eyJsb2NzdHJpbmciOiJjdGdBOzEwNTAuLjkwMDAiLCJ0cmFja0lEIjoiIiwiTmFtZSI6WyJFREVOLjEiXSwiTm90ZSI6WyJFZGVuIHNwbGljZSBmb3JtIDEiXSwiSUQiOlsiRURFTi4xIl19,5 eyJsb2NzdHJpbmciOiJjdGdBOzEwNTAuLjkwMDAiLCJ0cmFja0lEIjoiIiwiTmFtZSI6WyJFREVOLjEiXSwiTm90ZSI6WyJFZGVuIHNwbGljZSBmb3JtIDEiXSwiSUQiOlsiRURFTi4xIl19,5 eyJsb2NzdHJpbmciOiJjdGdBOzEwMDAuLjEwMDAiLCJ0cmFja0lEIjoiIiwiTmFtZSI6WyJGYWtlU05QIl0sIk5vdGUiOlsiVGhpcyBpcyBhIGZha2UgU05QIHRoYXQgc2hvdWxkIGFwcGVhciBhdCAxMDAwIHdpdGggbGVuZ3RoIDEiXSwiSUQiOlsiRmFrZVNOUDEiXX0=,14 eyJsb2NzdHJpbmciOiJjdGdBOzEwMDAuLjEwMDAiLCJ0cmFja0lEIjoiIiwiTmFtZSI6WyJGYWtlU05QIl0sIk5vdGUiOlsiVGhpcyBpcyBhIGZha2UgU05QIHRoYXQgc2hvdWxkIGFwcGVhciBhdCAxMDAwIHdpdGggbGVuZ3RoIDEiXSwiSUQiOlsiRmFrZVNOUDEiXX0=,14
```

Comment:
I think this is an older file? it uses an older base64 encoding. can it be removed?

Reply:
@TiaSFlores

Reply:
Yeah I can go through and make sure no old files are in the PR. This and the non assembly named meta.jsons are old files